docs/api-reference/dynamic-configuration-api.md

```diff
@@ -23,53 +23,591 @@ sidebar_label: Dynamic configuration
   ~ under the License.
   -->
 
-This document describes the API endpoints to retrieve and manage the dynamic configurations for the [Coordinator](../configuration/index.md#overlord-dynamic-configuration) and [Overlord](../configuration/index.md#dynamic-configuration) in Apache Druid.
+This document describes the API endpoints to retrieve and manage dynamic configurations for the [Coordinator](../design/coordinator.md) and [Overlord](../design/overlord.md) in Apache Druid.
+
+In this topic, `http://ROUTER_IP:ROUTER_PORT` is a placeholder for your Router service address and port.
+Replace it with the information for your deployment.
+For example, use `http://localhost:8888` for quickstart deployments.
 
 ## Coordinator dynamic configuration
 
-See [Coordinator Dynamic Configuration](../configuration/index.md#dynamic-configuration) for details.
+The Coordinator has dynamic configurations to tune certain behavior on the fly, without requiring a service restart.
+For information on dynamic configuration spec properties, see [Coordinator dynamic configuration](../configuration/index.md#dynamic-configuration).
+
+### Get Coordinator dynamic configuration
```

Comment:
Since these are all in the "Coordinator" heading, what do you think about removing it from the subsection headings to make it a bit easier to skim? So for example, something like:
* Get dynamic configuration
* Update dynamic configuration
* Get dynamic configuration history

Current preview:
![image](https://github.com/apache/druid/assets/7747997/56bb8f3a-12f8-4fbe-8ba4-e9f89e4ed7fc)

Reply:
That's a great idea.